ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,30 @@
+name: 🐛 Bug
+description: Report an issue to help improve the project.
+labels: ["🛠 goal: fix"]
```

These labels do not exist on this repository. 👀
```suggestion
labels: ["bug"]
```

---

Okay.

---

@Panquesito7 these are fallback files for other repos 🙂